common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -41,19 +41,43 @@ public ConceptService(EntityManager entityManager) {
   public Slice<Concept> searchConcepts(String query,
       StandardConceptFilter standardConceptFilter, List<String> vocabularyIds,
       List<String> domainIds, int limit) {
+
+      //String[] keywords=query.split("\\s*(=>|,|\\s)\\s*");
+      String[] keywords=query.split("[,+\\s+]");
+      for(int i=0;i<keywords.length;i++){
+          String key=keywords[i];
+          if(key.length() < 3){
+              key="\""+key+"\"";
+              keywords[i]=key;
+          }
+      }
+
+      String query2=new String();
+      for(String key:keywords){
+          if(query2.length()==0){
+              query2="+"+key;
+          }else if(key.contains("\"")){
```

Comment:
So if I type in:

hello "foo bar" baz

This will I believe produce:

+hello"foo bar"+baz

Is that correct? Is that what MATCH expects?